prepare_ubuntu16.04.yml

```diff
@@ -2,6 +2,5 @@
 - name: Prepare Ubuntu 16.0x with Python 2
   hosts: all
   gather_facts: false
-  tasks:
-    - name: Install python2
-      raw: sudo apt-get -y install python-simplejson
+  roles:
+    - common
```

Comment:
Why a separate playbook for the common role ? 

This playbook was used to install python2.7 on ubuntu server because it wasn't available